about.html

```diff
@@ -0,0 +1,108 @@
+            <img src="./image/iso-class.webp" alt="iso bim class">
+        </section>
+        <hr>
+        <section class="courses">
+            <div class="courses-header">
+                <h3>Course Categories</h3>
+                <hr>
+                <p>Our training courses are available in multiple formats, including virtual classroom.</p>
+            </div>
+            <div class="course-items">
+                <article class="course-item">
+                    <div class="overlay"></div>
+                    <div class="text">
+                        <h4 class="course-h">Building Information Modelling</h4>
+                        <p class="course-description">Building Information Modelling (BIM) and working with digital twins is becoming an increasingly important part of project design, improving the efficiency of works and easing building maintenance requirements after construction is complete.</p>
+                    </div>
+                        <img src="https://auuiyqpdxq.cloudimg.io/v7/https://www.icetraining.org.uk:443/getmedia/925f25fe-93ff-4d15-a92c-797cc88b93a7/BIM.jpg?width=936" alt="Building Information Modelling (BIM)"> 
+                </article>
+                <article class="course-item">
+                    <div class="overlay"></div>
+                    <div class="text">
+                        <h4 class="course-h">Accredited Project Managemen</h4>
+                        <p class="course-description">These online training courses provide construction professionals with the key project management skills needed for the delivery of successful projects. As well as the two longer courses preparing for exams, we offer a number of bite-size eLearning modules for quick information on specific topics.</p>
+                    </div>
+                    <img src="https://auuiyqpdxq.cloudimg.io/v7/https://www.icetraining.org.uk:443/getmedia/aa26ec9e-790a-4628-b1f6-e0a385d9fec6/collaborative-working.jpg?width=936" alt="Accredited Project Management">
+                </article>
+            </div>
+        </section>
+        <section class="partner">
+            <h3 class="partner-h">Partner</h3>
+            <hr>
+            <div class="partner-logo">
+                <ul>
+                    <li><img src="./image/logo-partner/allstate-logo.svg" alt="allstate-logo"></li>
+                    <li><img src="./image/logo-partner/asus-logo.svg" alt="asus-logo"></li>
+                    <li><img src="./image/logo-partner/atlassian-logo.svg" alt="atlassian-logo"></li>
+                    <li><img src="./image/logo-partner/autodesk-logo.svg" alt="autodesk-logo.svg"></li>
+                    <li><img src="./image/logo-partner/chase-logo.svg" alt="chase-logo"></li>
+                </ul>
+            </div>
+        </section>
```

Comment:
- [ ] I think it would be better if you remove the Partner section from the **About** page as it doesn't exist in the template design of the **About** page